core/src/main/java/org/pegdown/ParserWithDirectives.java

```diff
@@ -162,7 +162,7 @@ public Node parseBlockContents(Var<Integer> markerLength, StringBuilderVar block
         // recursively parse inner contents
         Context<Object> context = getContext();
         Integer length = markerLength.get();
-        String contents = block.getString();
+        String contents = "\n" + block.getString() + "\n";
```

Comment:
@sirthias Do you have an opinion on this topic as we've just merged #58, which I think assumes `@@@div` to render as `<div>content</div>` as your spec asserts, as opposed to `<div><p>content</p></div>`.